docs/glossary.rst

```diff
@@ -0,0 +1,400 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+This Reference article is a glossary. It is meant to provide a general overview
+of terms common to CrateDB Cloud. Because it forms a counterpart to the
+Glossary for CrateDB, we mainly cover terms here that are specific or primary
+to CrateDB Cloud; terms particular to the whole CrateDB architecture will
+appear in the CrateDB Glossary instead. We hope this Glossary will help you
+understand the basic meaning of concepts in the Console, Documentation, the
+CLI, and other Cloud-related sources. It is not meant to be a general guide to
+IT or IIoT terminology.
+
+In entries, terms used that have entries of their own are cross-referenced
+through links. The entries are presented in alphabetic order.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _glossary-audit-log:
+
+Audit log
+---------
+
+The Audit Log registers and displays all operations associated with a
+particular :ref:`organization<glossary-org>`. This includes operations on
+:ref:`users<glossary-user>`, on :ref:`clusters<glossary-cluster>`, on
+:ref:`projects<glossary-project>`, and on :ref:`consumers<glossary-consumer>`.
+The Audit Log can be found in the rightmost tab of the Organization overview
+page in the CrateDB Cloud :ref:`Console<glossary-console>`. Only an
+*organization admin* has access to the Audit Log.
+
+
+.. _glossary-amazon-cognito:
+
+Amazon Cognito
```

Comment:
not sure if we need amazon cognito in the glossary. 
In general we should refer to the "cognito signup" as username+password authentication.
Cognito is just the technical implementation of it.

Reply:
Cognito was suggested by Stephi - I'm happy to get rid of it though

Reply:
I'm fine with getting rid of it :)